src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
Yeah that's a good point @scottsb. I've gotten bogged down in a pretty intense closure hell when debugging JS exceptions/`Error`'s. Closure? Anonymous function? WHICH ONE?!@! 

Question is -- is that really necessary here? Then again it's not a big deal either way.